### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
----

### Before the change?

* typos https://github.com/jsoref/plugin-paginate-graphql.js/actions/runs/7950796206#summary-21703502067

### After the change?

* https://github.com/jsoref/plugin-paginate-graphql.js/actions/runs/7950796358

### Pull request checklist
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

```diff
-const { paginateGraphql } = require("@octokit/plugin-paginate-graphql");
+const { paginateGraphQL } = require("@octokit/plugin-paginate-graphql");
```

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [x] Yes
- [ ] No

----

I'm happy to drop that one change. There were enough typos in the neighborhood that I'd like to make the argument that it's better to properly spell GraphQL than to inconsistently spell it including with typos.

### CODE REVIEW

File: test/paginate.test.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ describe("pagination", () => {
       `
       query paginate($cursor: String, $organization: String!) {
         repository(owner: $organization, name: "rest.js") {
-          issues(first: 10, after: $curosr) {
+          issues(first: 10, after: $cursor) {
```
Comment (jsoref) [inline test/paginate.test.ts:109]: I don't think this test was testing what it thought it was testing.

File: test/typescript-validate.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export async function typedIterator() {
   }
 }
 
-export function pageInfoBackwaredExported(): PageInfoBackward {
+export function pageInfoBackwardExported(): PageInfoBackward {
```
Comment (jsoref) [inline test/typescript-validate.ts:40]: this is a test, but, um... I'm not sure what it was doing either.

**Review by kfcampbell — APPROVED** [review]
I'm :+1: on this PR, the breaking change is minimal and easy to adjust for and the other changes are helpful. I'll hold off on merging for now as I know @wolfy1339 is doing a lot of (excellent) work releasing in our JS ecosystem right now and I don't want to step on toes. If Wolfy is good with a release here, I can merge/release.

**Comment by wolfy1339** [comment]
I'm going to put this into the `beta` branch and just lump it in with my other changes

**Pull request merged**